WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -284,12 +276,7 @@ extension OrderListViewModel {
                     return .none
                 }
 
-                switch enrolledState {
-                case (.completed):
-                    return .simplePaymentsEnabled
-                default:
-                    return .none
-                }
+                return .simplePaymentsEnabled
```

Comment:
nit: Since simple payments is enabled for everyone, it might be a little clearer to call this case `simplePayments` instead of `simplePaymentsEnabled` now. Not a big deal, though!